test/jdk/java/nio/channels/Channels/TransferTo.java

```diff
@@ -169,6 +183,37 @@ public void testMoreThanTwoGB() throws IOException {
         }
     }
 
+    /*
+     * Special test whether selectable channel based transfer throws blocking mode exception.
+     */
+    @Test
+    public void testIllegalBlockingMode() throws IOException {
+        Pipe pipe = Pipe.open();
+        try {
+            // testing arbitrary input (here: empty file) to non-blocking selectable output
+            try (FileChannel fc = FileChannel.open(Files.createTempFile(null, null));
```

Comment:
As mentioned in a previous PR review, please create the temp file in the test directory and provide a prefix.

Reply:
With "the test directory", you mean the current working directory `"."`?

Reply:
Done in https://github.com/openjdk/jdk/pull/5179/commits/807139e0daf0647a868b8eab45bfb5efac944410.